src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -121,4 +124,67 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
         }
         return tagSet;
     }
+
+    //=========== LeavesBook ================================================================================
+
+    /**
+     * Parses a {@code String title} into a {@code String}.
+     * Leading and trailing whitespaces will be trimmed.
+     */
+    public static String parseTitle(String title) {
+        requireNonNull(title);
+        String trimmedTitle = title.trim();
+        return trimmedTitle;
+    }
+
+    /**
+     * Parses a {@code String dateStart} into an {@code Date}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code address} is invalid.
+     */
+    public static Date parseDateStart(String start) throws ParseException {
```

Comment:
Is there a reason why parseDateStart and parseDateEnd are two separate methods instead of just parseDate + having the user perform the necessary checks? Alternatively, we could create a DateRange object consisting of two dates and perform the checks at that level.